cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-search-performance/index.asciidoc

```diff
@@ -105,6 +105,34 @@ ON (r.propertyName_1,
 
 Index provider and configuration can be specified using the `OPTIONS` clause.
 
+| [source, cypher, role=noplay]
+----
+CREATE RANGE INDEX [index_name] [IF NOT EXISTS]
+FOR (n:LabelName)
+ON (n.propertyName_1[,
+    n.propertyName_2,
+    …
+    n.propertyName_n])
+[OPTIONS "{" option: value[, ...] "}"]
+----
+| Create a range index on nodes, either on a single property or composite.
```

Comment:
I would assume it applies to both since it has to do with what the indexes support on kernel level (correct me if I'm wrong @burqen or @emmaholmbergohlsson).

But yes we should probably mention that in the section (which technically talks about planning and using the indexes so it is the planner teams area and would also depend on if they plan the range indexes with the extra filters that I once upon a time added for the btree ones)

Reply:
Yes, both btree and range supports composite index.

Reply:
with the same limitations of not handling some predicates (multiple range predicates for example) I assume